sw/device/lib/testing/alert_handler_testutils.c

```diff
@@ -69,19 +69,17 @@ void alert_info_to_string(const alert_info_t *info) {
            info->class_accum_cnt[0], info->class_accum_cnt[1],
            info->class_accum_cnt[2], info->class_accum_cnt[3]);
   LOG_INFO("loc_alert_cause=0x%x", info->loc_alert_cause);
-  char cause_bits[81];
-  cause_bits[80] = '\0';
-  int string_index = 0;
+  int set_count = 0;
```

Comment:
The previous way to show the alert_cause bits was pretty bad :(
Showing the on bits only is much better, since most of the time very few bits are on.

Reply:
Nit: maybe useful to add this comment directly into the source code.

Reply:
Good idea. Done.

Reply:
Thanks!